### DISCUSSION TITLE
suggestions!

### DISCUSSION THREAD

**Discussion by Patchwxrk** [keiwando/toonsquid#22] [Ideas]
been using the app for around a week now and im in love, a few suggestions though

-reference layers
-Magic wand
-more intense stabilizer 
-ability to listen to music while working
-copy-paste
-when you pause your animation it takes you back to the frame you were on before

**Comment by chandanitis** [comment]
Is there a possibility of adding a multi-plane camera which can produce parallax effect?

**Reply by keiwando** [reply]
Yes, this is definitely something that I would like to look into in the future!

**Reply by chandanitis** [reply]
Thank you so much Kaiwando, I am super excited.

**Comment by keiwando** [comment]
I'm very glad to hear that you are enjoying ToonSquid! 
And thank you very much for the suggestions. All of these are already on my list of features to add or look into at some point in the future. 
I can already say that a higher brush smoothing setting, the ability to listen to music in the background even if you have audio layers and an option to jump back to the old frame when you hit pause will be part of the next (1.6) update (among other features) which should hopefully be ready relatively soon. Copy and paste for selection regions of a pixel layer and reference layers for the fill tool will most likely follow with the update after that.

**Reply by Patchwxrk** [reply]
great, can not wait!

**Reply by keiwando** [reply]
The 1.6 update is out now
